backend/database/migrations/2021_12_14_095332_create_lessons_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateLessonsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('lessons', function (Blueprint $table) {
+            $table->id();
+            $table->string('word');
+            $table->foreignId('category_id')
+                ->references('id')
+                ->on('categories');
+            $table->timestamps();
```

Comment:
You may also apply cascade onDelete. So that when category record id deleted it would automatically delete the child records related to the table, example you delete category 1, all leesson that have FK category_id of 1 will also be deleted.
```
->on('categories')
->onDelete('cascade');
```

Reply:
na miss sad diay ni nako. thank you sir 🙇‍♀️🙌